src/application/view_rankings.c

```diff
@@ -0,0 +1,1153 @@
+#include "application/view_rankings.h"
+
+#include "global.h"
+
+#include "constants/ranking.h"
+
+#include "application/guinness.naix"
+#include "msgdata/msg.naix"
+#include "msgdata/msg/msg_0421.h"
+
+#include "bg_window.h"
+#include "font.h"
+#include "gf_gfx_loader.h"
+#include "math_util.h"
+#include "msgdata.h"
+#include "obj_char_transfer.h"
+#include "player_data.h"
+#include "render_window.h"
+#include "save_rankings.h"
+#include "system.h"
+#include "text.h"
+#include "unk_02005D10.h"
+#include "unk_02009D48.h"
+#include "unk_0200A090.h"
+#include "unk_0200ACF0.h"
+#include "unk_0200B150.h"
+#include "unk_0200FA24.h"
+#include "unk_02022588.h"
+#include "unk_02023694.h"
+#include "unk_02097D3C.h"
+#include "vram_transfer_manager.h"
+#include "yes_no_prompt.h"
+
+#define min(a, b) ((a) <= (b) ? (a) : (b))
```

Comment:
should we define this in a header instead? Might be used elsewhere/just a nice function to have when modding

Reply:
thing is, we use `min` as a symbol in several places, so this would be a significant refactor